src/languages/es.ts

```diff
@@ -1926,6 +1927,24 @@ export default {
             welcomeNote: ({workspaceName}: WelcomeNoteParams) =>
                 `¡Has sido invitado a ${workspaceName}! Descargue la aplicación móvil Expensify en use.expensify.com/download para comenzar a rastrear sus gastos.`,
         },
+        distanceRates: {
```

Comment:
waiting for translations

Reply:
translations posted in slack

Reply:
I think there's a typo in the last one. It should be:

```
distanceRates: 'Tasas de distancia',
```